doc/HttpLuaModule.wiki

```diff
@@ -3495,7 +3495,7 @@ This method does not work in HTTP/2 requests yet.
 == ngx.req.get_method ==
 '''syntax:''' ''method_name = ngx.req.get_method()''
 
-'''context:''' ''set_by_lua*, rewrite_by_lua*, access_by_lua*, content_by_lua*, header_filter_by_lua*, balancer_by_lua*''
+'''context:''' ''set_by_lua*, rewrite_by_lua*, access_by_lua*, content_by_lua*, header_filter_by_lua*, log_by_lua*, balancer_by_lua*''
```

Comment:
@tokers Will you please also regenerate the Markdown file via the [wiki2markdown.pl](https://github.com/openresty/openresty-devel-utils/blob/master/wiki2markdown.pl) script? Thanks.

Reply:
@thibaultcha I have created another PR https://github.com/openresty/lua-nginx-module/pull/1395.

Reply:
I will close this Pull Request.